src/utils/github/getPRsToPaintPerSHAs.ts

```diff
@@ -50,9 +50,15 @@ export default async function getPRsToPaintPerSHAs({
     repoSource,
   });
 
-  if (foundPRs?.error === "no access_token") {
+  if (foundPRs.error === "no access_token") {
     return { errorText: "Not logged in" };
   }
+
+  // This handles GitHub, GitLab, and Bitbucket
+  if (foundPRs?.error?.includes("query limit reached")) {
+    return { errorText: `You've exceeded the number of free monthly queries. We invite you to upgrade your Watermelon plan here: https://app.watermelontools.com/billing` }
```

Comment:
I would add an item that navigates the user there but ok

Reply:
You mean an <a href> inside the string? (I tried it)

Or what do you mean? Sorry I don't get it